website/docs/r/wafv2_rule_group.html.markdown

```diff
@@ -492,11 +492,12 @@ The part of a web request that you want AWS WAF to inspect. Include the single `
 
 The `field_to_match` block supports the following arguments:
 
-~> **NOTE:** Only one of `all_query_arguments`, `body`, `method`, `query_string`, `single_header`, `single_query_argument`, or `uri_path` can be specified.
+~> **NOTE:** Only one of `all_query_arguments`, `body`, `cookies`, `method`, `query_string`, `single_header`, `single_query_argument`, or `uri_path` can be specified.
```

Comment:
The same changes should be added to `website/docs/r/wafv2_web_acl.html.markdown`.

Reply:
Fixed